icons/treehouse.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Treehouse icon</title><path d="M20.537 4.118c-.806-.453-2.092.278-2.871 1.635L16.25 8.215a3.104 3.104 0 0 0 .21 3.18l.041.062c.653.94 1.535 1.808 1.823 2.118a1.613 1.613 0 0 1-.739 2.654 1.603 1.603 0 0 1-2.025-1.747c.045-.35-.067-.927-.574-1.489-.506-.563-1.54.5-1.874 1.61l-.016.061c-.334 1.094-.546 2.05-.482 2.143.037.06.072.12.105.182a1.81 1.81 0 0 1-3.196 1.701 1.806 1.806 0 0 1 .747-2.446l.121-.061c.065-.03.26-.486.423-1.032l.301-.987c.019-.047.033-.107.045-.168l.897-3.19-.957 1.96c-.112-.363-.3-.38-.709-.091-.243.183-.653.531-.85.669-.365.273-.685.788-.851 1.109a1.313 1.313 0 0 1-.41.5c-.684.564-1.687.456-2.234-.227a1.591 1.591 0 0 1 .912-2.552c.409-.092 1.777-.927 2.596-1.52.152-.106.274-.197.38-.304l2.203-1.67-1.914 1.032s-.196-.016-.426.017c-.698.075-1.428.182-1.564.35a.999.999 0 0 1-.29.272c-.637.456-1.519.32-1.989-.317A1.437 1.437 0 0 1 6.29 8.04c.259-.183.577-.274.865-.274.518.016 1.87.29 2.993.092l.288-.045c1.14-.196 2.476-1.186 3.024-2.187l1.184-2.067c.653-1.139.608-2.384-.105-2.795l-1.323-.76c-.653-.363-1.715-.363-2.354 0L2.004 4.97C1.337 5.319.805 6.23.805 6.975v9.744c0 .744.532 1.656 1.178 2.02l8.85 4.983c.652.365 1.716.365 2.354 0l8.826-4.983c.653-.368 1.184-1.276 1.184-2.02v-9.76c0-.744-.531-1.653-1.169-2.02l-1.46-.823"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Treehouse icon</title><path d="M20.4431 4.3385c-.7972-.448-2.0692.275-2.8397 1.6172L16.2028 8.391a3.0702 3.0702 0 00.2077 3.1454l.0405.0614c.646.9298 1.5183 1.7883 1.8032 2.095a1.5955 1.5955 0 01-.731 2.625 1.5856 1.5856 0 01-2.003-1.728c.0446-.3461-.0662-.9168-.5677-1.4727-.5005-.557-1.5232.4945-1.8536 1.5925l-.0158.0603c-.3304 1.082-.54 2.0277-.4768 2.1197.0366.0593.0712.1187.1039.18A1.7905 1.7905 0 019.549 18.752a1.7864 1.7864 0 01.7388-2.4194l.1197-.0603c.0643-.0297.2572-.4808.4184-1.0208l.2977-.9763c.0188-.0465.0327-.1058.0445-.1661l.8873-3.1553-.9466 1.9386c-.1108-.359-.2967-.3758-.7013-.09-.2403.181-.6459.5253-.8407.6617-.361.27-.6776.7795-.8418 1.097a1.2987 1.2987 0 01-.4055.4945c-.6766.558-1.6687.451-2.2097-.2245a1.5737 1.5737 0 01.902-2.5242c.4046-.091 1.7577-.917 2.5678-1.5035.1504-.1048.271-.1948.3759-.3007l2.179-1.6518-1.8932 1.0208s-.1938-.0159-.4213.0168c-.6904.0742-1.4125.18-1.547.3462a.9881.9881 0 01-.2868.269 1.4147 1.4147 0 01-1.9674-.3135 1.4214 1.4214 0 01.3323-1.9723c.2562-.181.5708-.271.8556-.271.5124.0158 1.8497.2868 2.9605.091l.2848-.0446c1.1276-.1938 2.449-1.173 2.9911-2.1632l1.1711-2.0445c.646-1.1266.6014-2.358-.1038-2.7646L13.2018.2693c-.646-.359-1.6964-.359-2.3284 0l-8.7616 4.912C1.452 5.5265.9258 6.4275.9258 7.1644v9.638c0 .736.5262 1.638 1.1652 1.998l8.7537 4.9288c.645.361 1.6973.361 2.3284 0l8.73-4.9288c.6459-.364 1.171-1.262 1.171-1.998V7.1486c0-.7359-.5251-1.635-1.1562-1.998l-1.4441-.814"/></svg>
```

Comment:
Looks perfect! 🎉
Any chance of getting it down to 3 decimals?

Reply:
Done 💪🏼